### PR TITLE
Tuned shooter to decent accuracy

### PR DESCRIPTION
**Pull request by LaraLim**
- 1300 RPM 
- 30% feeder output
- P = 0.0025
- I = 0.0
- D = 0.004
- F = 0.0002

### CODE REVIEW

**Review by nicojopr — APPROVED** [review]

File: src/main/java/frc/robot/commands/VisionAssistedShooter.java [hunk 1]
```diff
@@ -1,3 +1,4 @@
+package frc.robot.commands;
```
Comment (georgetzavelas) [inline src/main/java/frc/robot/commands/VisionAssistedShooter.java:1]: Do you know why this line is needed when nothing else changed in the file?

**Review by georgetzavelas — COMMENTED** [review]
My favourite question, can we make these into constants? :)

File: src/main/java/frc/robot/commands/VisionAssistedShooter.java (hunk 1 above)
Comment (LaraLim) [reply src/main/java/frc/robot/commands/VisionAssistedShooter.java:1]: When I pulled from master it was somehow missing, so it caused an error.

**Review by Zinka010 — APPROVED** [review]

**Comment by georgetzavelas** [comment]
Maybe we can circle back when people have more time and make these into constants based on my comment? :)

**Pull request merged**